docs/api/field_type_reference.md

```diff
@@ -412,6 +412,29 @@ $countryValue = new Country\Value(
 );
 ```
 
+!!! dxp
+
+    ## DAM Field Type
+
+    This Field Type stores asset information from a DAM system.
+    It replaces the [ImageAsset Field Type](#imageasset-field-type).
+
+    | Name   | Internal name | Expected input type |
+    |--------|---------------|---------------------|
+    | `Image Asset` | `ezimageasset`      | mixed             |
+
+    #### Value object
+
+    ###### Properties
+
+    Value object of `dam` contains the following properties:
```

Comment:
```suggestion
    The `Value` object of the `dam` field contains the following properties:
```
W obrębie tego jednego artykułu są liczne wersje tego samego zwrotu. Może przy okazji (niekoniecznie tej) można by wybrać najbardziej poprawną i uspójnić to? (W przyszłości mniej zapłacimy za tłumaczenie).

Reply:
True, but, again we need to introduce consistency for all branched, so will have to be done in a separate PR.
And it's a ["Value object"](https://en.wikipedia.org/wiki/Value_object) as a fixed term.